chsdi/models/vector/bafu.py

```diff
@@ -821,30 +821,34 @@ class GewaesserschutzBadewasserqualitaet(Base, Vector):
 
 
 class AmG(Base, Vector):
-    __tablename__ = 'am_g'
+    __tablename__ = 'amphibien_wanderobjekte'
     __table_args__ = ({'schema': 'bundinv', 'autoload': False})
     __bodId__ = 'ch.bafu.bundesinventare-amphibien_wanderobjekte'
     __template__ = 'templates/htmlpopup/bundinv_amphibien_w.mako'
-    __label__ = 'am_g_name'
-    id = Column('am_g_obj', Unicode, primary_key=True)
-    am_g_name = Column('am_g_name', Unicode)
+    __label__ = 'name'
+    id = Column('objnummer', Unicode, primary_key=True)
```

Comment:
objectnummer is not an integer or float?

Reply:
Its a String because the object numbers include the abbreviation of the cantons, e.g. "AG31"